app/assets/stylesheets/orders.css.scss

```diff
@@ -3,6 +3,8 @@
   margin-top: 10%;
   margin-left: auto;
   margin-right: auto;
+  border: solid 2px #363636;
```

Comment:
Color literals like `#363636` should only be used in variable declarations; they should be referred to via variable everywhere else.